packages/dashboard/src/components/Card/CardPage.tsx

```diff
@@ -4,10 +4,18 @@ import { SmallCardBasicExample } from './examples/Card.Small.Basic.Example';
 import { MediumTallCardBasicExample } from './examples/Card.MediumTall.Basic.Example';
 import { MediumWideCardBasicExample } from './examples/Card.MediumWide.Basic.Example';
 import { LargeCardBasicExample } from './examples/Card.Large.Basic.Example';
+import { DonutAndPieChartExample } from './examples/DonutAndPieChartExample';
+import { LineAndVerticalBarChartExample } from './examples/LineAndVerticalBarChartExample';
+import { MultiCountExample } from './examples/MultiCountExample';
+import { StackedBarChartExample } from './examples/StackedBarChartExample';
```

Comment:
nit: Can we rename these files to follow the standard Card.Chart.Donut.Example, Card.Chart.Bar.Example, Card.Small.MultiCount.Example, Card.Chart.StackedBar.Example

Reply:
done